tokens.xml

```diff
@@ -2049,7 +2049,7 @@ Whenever Glyph Keeper becomes the target of a spell or ability for the first tim
         </card>
         <card>
             <name>Graveborn</name>
-            <set picURL="http://i167.photobucket.com/albums/u148/Ruja/Magic%20Tokens/Coldsnap/5.jpg">CSP</set>
+            <set picURL="https://i.imgur.com/1PwSeto.jpg">CSP</set>
```

Comment:
New token is missing "haste"!

Reply:
That's consistent with Wizard's current token printing policy.

Reply:
They don't print `haste` any longer on tokens?
I guess that's the only keyword they don't print?

I must have missed that...

Reply:
When it is the only keyword on a token, they won't print it. This has been the case since at least Avacyn Restored, perhaps earlier.